src/ProgressOnderwijsUtils/Extensions/StringExtensions.cs

```diff
@@ -77,10 +75,11 @@ public static string Replace(this string s, [NotNull] IEnumerable<KeyValuePair<s
         /// 'S-GRAVENHAGE -> 's-Gravenhage
         /// </summary>
         [Pure]
-        public static string ToCamelCase(this string s)
+        [return: NotNullIfNotNull("s")]

```

Comment:
yuk. Dus nu moeten we dit in sync houden?

Reply:
blijkbaar